test/regress/cli/regress1/sets/relation_project2.smt2

```diff
@@ -0,0 +1,12 @@
+(set-logic HO_ALL)
+(set-info :status sat)
+(set-option :fmf-bound true)
+(set-option :uf-lazy-ll true)
+
+(declare-fun A () (Relation String String))
+(declare-fun B () (Relation String String))
+
+(assert (distinct A (as set.empty (Relation String String))))
+(assert (= B ((_ rel.project 1 0) A)))
+(assert (set.member (tuple "y" "x") B))
+(check-sat)
```

Comment:
cvc5 returns unknown answer for the the bag version of this example. 
That is why it is not included in this PR

Reply:
Removing (distinct A bag.empty) was enough for cvc5 to answer sat.